src/components/contacComponents/contactForm/ContactForm.jsx

```diff
@@ -1,12 +1,19 @@
+import React from "react";
 import "./contactForm.css";
 
 export default function ContactForm() {
   return (
     <div>
+      <div className="sectionTitle">Get in touch</div>
```

Comment:
"Get in touch" should be a header, not a div.
Also, make margin bigger below it :)